onnx/defs/operator_sets.h

```diff
@@ -0,0 +1,385 @@
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
```

Comment:
So this file should be autogenerated, it's so easy to forget to update when adding new schemas and also unfriendly for new contributors.

Reply:
In either case, the operators won't be available or get generated into documentation, so I don't think this will be missed.  

Not requiring code-gen steps is simplifying in itself.  While I can imagine other benefits if we combine code-gen with other refactoring, I don't want to block this PR on that larger project.

Reply:
This will be missed. We can't rely on reviewers to remember how many docs changes they need to see so the changes are legit, the only viable way to guard such boilerplate changes is adding a check in the CI. Also think about new coming/occasional contributors, it's impossible for them to remember.

Reply:
Our thinking was that exposing operators which were never tested was not as important, but I understand wanting to make this easier to discover.

I'll see if I can cleanly add check in debug mode that the number of registration macros usages matches the total of schema exposed by the iteration methods.  Does that sound reasonable?

Reply:
I've made that change.  This also fixes a pre-existing bug in assert implementations which caused them to always pass.